src/components/about/About.test.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import About from "./About";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+test("should render component", () => {
+  const history = createMemoryHistory({
+    initialEntries: ["/about"],
+  });
+  const { container } = render(
+    <Router history={history}>
+      <About />
+    </Router>
+  );
+
+  const iconButtons = container.querySelectorAll(".icon-button__text");
+
+  expect(container.querySelector(".navbar__title")?.textContent).toEqual(
+    "About"
+  );
+  expect(container.querySelector(".about__subtitle")?.textContent).toEqual(
+    "App Settings"
+  );
+  expect(iconButtons[0]?.textContent).toEqual("Change language");
+  expect(iconButtons[1]?.textContent).toEqual("Change Theme");
+  expect(iconButtons[2]?.textContent).toEqual("Contact me");
+  expect(iconButtons[3]?.textContent).toEqual("Source code");
+  expect(iconButtons[4]?.textContent).toEqual("Report bug");
+});
```

Comment:
if you add `  screen.logTestingPlaygroundURL()` to the test after you render the component.  You will be able to find all the elements that can be selected and tested against.

https://testing-playground.com/#markup=DwEwlgbgfKkAQGMA2BDAzmgvAIhQIwHsBXAF2xnAkVQxwDsUI8UAnOfYkgfS4adfLAU1dFmwBZImABCpEgTrT0AUwC0LAgRJw8chXDAIFq3SXl04fZix7MEAaxN662OCXxg6IZQA8cABlcNJGUcU3NXVjAUVXA0fBCQHAAzFCQ0ZVcQFHdVHIIAWyczBRwSFiJMinhkURxDY3CFHgaXGDQABxQLWtpsVwB3MBASAAscACYAFldR5TAAc1GSSZmYAHpO7o3Kdq6emjFJMAAVYgRRgCUwDo6Q9U0yDa26DZRqql6xK1YeEjASCFyABBQikYDrXYQqGUER9BCsEDsMHcLgoW6qDosZTJZTYugIZRoQSwr44DikHhoIh4f6AqrA25wADKyjMngWaGhkA+cLEFNR6I6mOxuPxhLQtmcxJgwjJEikshKihUDy0OmcBiMdGK5jgSAICPuGRCCHkNi4TRcbg8Xl8ASCBBCYWckRY0ViYHieESKTSGSyORi+SKVrKFSqsE+h3q2t1zS4rUELz5OEGwzGq1m8yWK2w00Em32Ox5UdT2Fa8bof18TwAwqNugtlPqm0QUM3udBgCn5cczkQLtdbvcNFpCy83rLy8cleYlBk1dorVrGq6bXhPN4-NhAnBgqFsFa3R64gllElsKl0pk4NlciGq+HKiSajGK3GrS1tcn9uX0yM4z5jMcBzIsyxZs8xZdry8qVl+XAkLW5ANk2LYnHMBTKDBPZ-n2UgDkONx3GoY5PBCk4Qu8OGQqWpLvgiLBIgKPACq+0Z1LgKKsdxlrSoIcrvrOzgLqRjwasqq46seG5bvau4njEZ4+hefo3oGD7yKGzjPreoyijgywkB0aAAFzrOsJBDGYeIAHRGAU6wABIEBUaAAOLEEg7HlvBzjfgov7dP+cBDIBWagTmEHARO0G0d29GcX5yo1j49YKO4ZpwFhOG9kJBHnFcxGjo8sXbFR074TIImqmREl6slESyXaO57lESleuel7XgGd5BnkWlPtg5QvqBBnYEZJnmesCwAqMNL2YUzmuUQHleesj4dAMKA+XBn7+YmP57MF8oAZmMWReBeYFlB5XxbB76NQmSFpeQzLECwhKIAQ3i5Xh+WnIVw4kUuZWvBVQgzoqNWLnVK5Pda7ibi1DrsO6HXer6V7+re97BoNYbDRGszjQUKBgEg8imaMK1oAsXkAAILGTFOLQUu2PftKWHYFx0HJxZ1AQWl25pBFFxTCb5JVz5ipU8lzKB0rnLkQCx-SdAOEUVI5ieOt3gxtJbdvdNG7EAA

..click on the element and you can copy the selector to test against.  

eg for example it shows you can use this  `screen.getByText(/change language/i)`  instead of a queryselector to get the icon test

Reply:
Such a wonderful suggestion. Thanks